service-loadbalancer/template.cfg

```diff
@@ -114,14 +110,14 @@ backend {{$svc.Name}}
     # http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#stick-table
     stick-table type ip size 100k expire 30m
     stick on src
-    {{range $j, $ep := $svc.Ep}}server {{$ep}} {{$ep}}
+    {{range $j, $ep := $svc.Ep}}server {{$ep}} {{$ep}} check port {{$svc.BackendPort}} inter 5
```

Comment:
WDYT about doing what you once suggested (make this go straight to the liveness probe of the pod)? just a suggestion, ok to punt